src/Classes/Factories/UserFactory/index.ts

```diff
@@ -9,37 +9,11 @@ export default class UserFactory {
     userData: UserData,
   ): User | Admin {
     let user;
-    const {
-      id,
-      username,
-      isAdmin,
-      email,
-      password,
-      articles,
-      articlesToApprove,
-      posts,
-    } = userData;
+    const { id, username, isAdmin, email, password } = userData;
```

Comment:
I suggest making UserFactory responsible only for creating User instance without DB connectors